src/Modules/OAuth/Backend/Actions/ModuleSettings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace ForkCMS\Modules\OAuth\Backend\Actions;
+
+use ForkCMS\Core\Domain\Header\Breadcrumb\Breadcrumb;
+use ForkCMS\Core\Domain\Header\FlashMessage\FlashMessage;
+use ForkCMS\Modules\Backend\Domain\Action\AbstractFormActionController;
+use ForkCMS\Modules\Backend\Domain\Action\ActionServices;
+use ForkCMS\Modules\OAuth\Domain\Settings\Command\UpdateModuleSettings;
+use ForkCMS\Modules\OAuth\Domain\Settings\SettingsType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class ModuleSettings extends AbstractFormActionController
+{
+    public function __construct(ActionServices $services, private readonly \ForkCMS\Modules\Extensions\Domain\Module\ModuleSettings $moduleSettings)
```

Comment:
Is die lijn niet te lang voor de codesteles?

Reply:
Niet logischer om die `ModuleSettings` te importeren via use statement?

Reply:
fixed ee48f6fd1acace8b314497421104cd0ea3226352